File: VonexDealer.API/Models/Order/Manufacturer.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace VonexDealer.API.Models.Order
{
    /// <summary>
    /// QA list of Manufacturers.
    /// </summary>
    public class Manufacturer
    {
        public int ManufacturerID { get; set; }
        public string Description { get; set; }
    }
}

File: VonexDealer.Vue/ClientApp/models/manufacturer.js
export class manufacturer {
    constructor({
        manufacturerID = 0,
        description = null
    } = {}) {
        this.manufacturerID = manufacturerID,
            this.description = description
    }

}

export function createManufacturer(data) {
    return Object.freeze(new manufacturer(data))
}
